packages/api/lib/testUtils.js

```diff
@@ -313,7 +313,7 @@ function fakeCumulusMessageFactory(params = {}) {
   return merge({
     cumulus_meta: {
       workflow_start_time: 122,
-      cumulus_version: '7.1.0',
+      cumulus_version: '9.2.0',
```

Comment:
This was causing an integration test to fail because it was skipping PG writes.

Reply:
:oof: